spec/smart_app_launch/discovery_group_spec.rb

```diff
@@ -49,6 +49,14 @@ def run(runnable, inputs = {})
       expect(result.result).to eq('pass')
     end
 
+    it 'sends an Accept Header with application/json' do
+      request = stub_request(:get, well_known_url)
+        .to_return(status: 200, body: well_known_config.to_json, headers: { 'Content-Type' => 'application/json' })
+      result = run(runnable, url: url)
+      
+      expect(a_request(:get, well_known_url).with(headers: {'Accept' => 'application/json'})).to have_been_made.once
```

Comment:
request = stub_request(:get, well_known_url)
        .with(headers: {'Accept' => 'application/json'})
        .to_return(status: 200, body: well_known_config.to_json, headers: { 'Content-Type' => 'application/json' })
      result = run(runnable, url: url)

      expect(request).to have_been_made.once

Reply:
I can make this change, but I don't think it's really necessary. This test is really just to ensure that we don't have a regression where Inferno stops sending an Accept header and there is another test that mocks the response and makes sure the tests pass.